config/locales/candidate_interface/new_references.yml

```diff
@@ -0,0 +1,119 @@
+          - You can also add more referees to increase your chances of getting a reference quickly.
+        cancelled: Training providers will not see any information about this reference request.
+        cancelled_at_end_of_cycle: The referee did not respond before courses closed.
+        email_bounced: The reference request never reached your referee.
+        not_requested_yet: We’ll contact your referees after you submit your application.
+        awaiting_reference_sent_less_than_5_days_ago: We’ve emailed your referee. Keep in touch with them to ensure they’re planning on giving a reference as soon as possible.
+        awaiting_reference_sent_more_than_5_days_ago: Your referee has not responded yet. Ask them if they got the email - it may have gone to junk or spam.
+      review:
+        more_than_two: More than 2 references have been given
+        need_two: You need to have received at least 2 references before marking this section complete
+        incorrect_number_selected: You need to have exactly 2 references selected before submitting your application
+      add_reference:
+        zero: Add reference
+        one: Add another reference
+        other: Add another reference
+      status:
+        first_line: '%{name} will not be asked to give you another reference.'
+        second_line: If you accept an offer, your previous reference will be shown to the provider.
+
+  activemodel:
+    errors:
+      models:
+        candidate_interface/add_another_referee_form:
+          attributes:
+            add_another_referee:
+              blank: Select if you want to add another referee
+        candidate_interface/reference/referee_type_form:
+          attributes:
+            referee_type:
+              blank: Choose a type of referee
+        candidate_interface/reference/referee_name_form:
+          attributes:
+            name:
+              blank: Enter your referee’s name
+              too_short: Your referee’s name must be %{count} characters or more
+        candidate_interface/reference/referee_email_address_form:
+          attributes:
+            email_address:
+              blank: Enter your referee’s email address
+              duplicate: Please give a different email address for each referee
+              own: Enter an email address that’s not your own
+        candidate_interface/reference/referee_relationship_form:
+          attributes:
+            relationship:
+              blank: Enter how you know this referee and for how long
+        candidate_interface/reference/submit_referee_form:
+          attributes:
+            submit:
+              blank: Choose whether to send your reference request now
+              incomplete: Add all of your referee’s details
+        candidate_interface/reference/candidate_name_form:
+          attributes:
+            first_name:
+              blank: Enter your first name
+            last_name:
+              blank: Enter your last name
```

Comment:
This should be no longer needed, as we don’t need to ask for the candidate’s name as part of the reference flow now.

```suggestion
```